File: aiJ-client/assets/Script/plazz/event/UserCertEvent.ts
import {AiJ} from "../../ws/AiJ";

export default class UserCertEvent extends AiJ.AiJEvent {


    certName: string;
    /**
     * 真实证件号
     */
    certCard: string;
    /**
     * 真实手机
     */
    certMobile: string;
    /**
     * 实名制类型
     */
    certType: string;

    constructor(certName: string, certCard: string, certMobile: string, certType: string) {
        super();
        this.mainType = 4;
        this.subType = 4;
        this.certName = certName;
        this.certCard = certCard;
        this.certMobile = certMobile;
        this.certType = certType;
    }
}

File: aiJ-plaza/src/main/java/com/xiyoufang/aij/plaza/event/UserCertEvent.java
package com.xiyoufang.aij.plaza.event;

import com.xiyoufang.aij.event.Event;

/**
 * Created by 席有芳 on 2019-03-03.
 * 用户实名
 *
 * @author 席有芳
 */
public class UserCertEvent extends Event {
    /**
     * 真实姓名
     */
    private String certName;
    /**
     * 真实证件号
     */
    private String certCard;
    /**
     * 真实手机
     */
    private String certMobile;
    /**
     * 实名制类型
     */
    private String certType;

    public String getCertName() {
        return certName;
    }

    public void setCertName(String certName) {
        this.certName = certName;
    }

    public String getCertCard() {
        return certCard;
    }

    public void setCertCard(String certCard) {
        this.certCard = certCard;
    }

    public String getCertMobile() {
        return certMobile;
    }

    public void setCertMobile(String certMobile) {
        this.certMobile = certMobile;
    }

    public String getCertType() {
        return certType;
    }

    public void setCertType(String certType) {
        this.certType = certType;
    }
}
